File: chroma-league/src/test/java/com/bonepl/chromaleague/hud/animations/BaronBuffBackgroundAnimationTest.java
package com.bonepl.chromaleague.hud.animations;

import com.bonepl.chromaleague.hud.AnimationTester;
import org.junit.jupiter.api.Test;

class BaronBuffBackgroundAnimationTest {

    @SuppressWarnings("JUnitTestMethodWithNoAssertions")
    @Test
    void playBaronBuffBackgroundAnimation() {
        new AnimationTester().testAnimation(new BaronBuffBackgroundAnimation(), 100);
    }
}

File: test/hud/animations/BaronBuffBackgroundAnimation.test.ts
import {describe, test} from 'vitest';
import {AnimationTester} from '../../AnimationTester.js';
import {BaronBuffBackgroundAnimation} from '../../../src/hud/animations/BaronBuffBackgroundAnimation.js';

/**
 * Integration test for baron buff background animation on actual hardware.
 */
describe('BaronBuffBackgroundAnimation Integration', () => {
    test('playBaronBuffBackgroundAnimation', async () => {
        await new AnimationTester().testAnimation(new BaronBuffBackgroundAnimation(), 100);
    }, 30000);
});
